### ISSUE TITLE
关于实时性的问题

### ISSUE THREAD

**Issue by huihui1114** [Xiaobin-Rong/gtcrn#45]
问题：我在使用您开源的代码和预训练模型进行推理的时候，我把输入模型的帧数设置为8,16,32,64,128等，发现模型在帧数小于64时效果会有影响，按照帧长512，帧移256，采样率16K，时延达到了64*256/16ms=1024ms,延迟太大了。
原因：我去查找了代码发现模型中有批量归一化模块nn.BatchNorm2d导致每帧信号收到上下帧之间的影响。
疑问：请问您如何解决这个问题，会有哪些影响？

**Comment by Xiaobin-Rong** [comment]
请问您使用的是常规推理版本还是流式推理版本？我猜您应该使用了常规推理版本，该版本是一种非实时的处理（即对一整段长音频来处理）。使用流式推理是可以实时推理的，模型的延时是2帧，而不是64帧。
另外，BatchNorm应该使每帧信号受到相邻帧的影响，因为BatchNorm的参数在训练完毕之后就固定下来了，和时间维度是无关的。
